src/99-appendices/03-hed.md

```diff
@@ -3,207 +3,175 @@
+appearing in the `HED` column of `_events.tsv` files.
+The HED strings from all sources are concatenated to form the final
+event annotations.
+
+Annotations placed in sidecars are preferred to those placed
+directly in the `HED` column, because they are simpler, more compact,
+more easily edited, and less prone to inconsistent annotation.
+
+## HED and the BIDS inheritance principle
+
+Most studies have event files whose columns contain categorical and
+numerical values that are similar across the recordings in the study.
+Ideally, these columns should be annotated in a single `_events.json` sidecar
+placed at the top level in the dataset.
+
+If some recordings in the dataset have a column whose values deviate from a
+standard pattern, then the annotations for that column MUST be placed in
+sidecars at lower levels in the hierarchy.
+According to the BIDS [inheritance principle](../02-common-principles.md#the-inheritance-principle),
+once a column key in a sidecar (the column name found in the `_events.tsv` files)
+is set, it cannot be unset by a sidecar appearing at a lower level in the dataset hierarchy.
+
+## HED schema versions
 
 The HED vocabulary is specified by a HED schema,
 which delineates the allowed HED path strings.
-By default, BIDS uses the latest HED schema available in the
-[hed-specification](https://github.com/hed-standard/hed-specification/tree/master/hedxml) repository
-maintained by the hed-standard group.
-
-You can override the default by providing a specific HED version number in the
-`dataset_description.json` file using the `HEDVersion` field.
-The preferred approach is to validate with the latest version (the default),
-but to use the `HEDVersion` field to specify which version was used for later reference.
+The version of HED used in tagging a dataset should be provided in the `HEDVersion`
+field of the `dataset_description.json` file located in the dataset root directory.
 
 Example: The following `dataset_description.json` file specifies that
-`HED7.1.1.xml` from the [hed-specification](https://github.com/hed-standard/hed-specification/tree/master/hedxml) repository
-should be used to validate the study event annotations.
+[`HED8.0.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.0.0.xml)
+file from the `hedxml` directory of the
+[`hed-specification`](https://github.com/hed-standard/hed-specification)
+repository on GitHub should be used to validate the study event annotations.
 
 ```JSON
 {
-  "Name": "The mother of all experiments",
-  "BIDSVersion": "1.4.0",
-  "HEDVersion": "7.1.1"
+  "Name": "A great experiment",
+  "BIDSVersion": "1.6.0",
+  "HEDVersion": "8.0.0"
 }
 ```
+
+If you omit the `HEDVersion` from the dataset description file
+or are running the BIDS validator behind a firewall and cannot access GitHub,
+the HED validation will fall back to the latest version of the HED schema,
+which it stores internally.
```

Comment:
I removed the technical detail on the firewall ... if you absolutely need it, then I would suggest putting it as a minor fact or in parentheses, after the important things have been stated.

```suggestion
If you omit the `HEDVersion` field from the dataset description file,
any present HED information will be validated using the latest version of the HED schema,
which is bound to result in problems.
Hence, it is strongly RECOMMENDED to supply the `HEDVersion` field when using HED
in a BIDS dataset.
```